src/ipc/byt-ipc.c

```diff
@@ -58,13 +58,13 @@ static void do_notify(void)
 	uint32_t flags;
 	struct ipc_msg *msg;
 
-	tracev_ipc("Not");
-
 	spin_lock_irq(&_ipc->lock, flags);
 	msg = _ipc->shared_ctx->dsp_msg;
 	if (msg == NULL)
 		goto out;
 
+	trace_ipc("ipc: not rx -> 0x%x", msg->header);
+
```

Comment:
for my education, have we stopped using the 3 letter code for errors? If yes, we might want to revisit all the drivers to provide more meaningful information that Sst or ssr for SSP start / resume

Reply:
yes, that's something we have todo - I'm hoping this can be the example to copy.